src/main/no/nav/pto/veilarbfilter/HttpHandler.kt

```diff
@@ -33,15 +33,24 @@ fun createHttpServer(
 
 
     install(Authentication) {
-        jwt {
-            authHeader(JwtUtil.Companion::useJwtFromCookie)
+        jwt("AzureAD") {
+            skipWhen { applicationCall -> applicationCall.request.cookies["isso-idtoken"] == null }
```

Comment:
Siden vi kommer til å bruke "isso-idtoken" og "ID_token" flere ganger i koden, så tenker jeg det kunne vært greit å lage en konstant for hver av de